telomerecat/telbam2length.py

```diff
@@ -917,11 +1036,29 @@ def run(
         inserts_path, insert_length_generator, vital_stats
       )
 
+      # create file name for current error profile
+      if error_path is not None and global_error_profile is None:
+        current_error_path = error_path
+      else:
+        current_error_path = None
+
       read_type_counts = self.__get_read_types__(
-        sample_path, vital_stats, self.total_procs, trim, seed_randomness
+        sample_path, vital_stats, self.total_procs, trim, seed_randomness, names[i], error_profile=global_error_profile, error_path=current_error_path, error_list=error_list
```

Comment:
Another very long line. Please shorten

Reply:
Throughout the code there are quite a few long lines. I know this a bit nitpicky but PEP8 style guide says lines should be kept neat. It helps readability in code a lot, I find. 

It would be good to keep lines ~100chars throughout please